runtime/compiler/trj9/ilgen/Walker.cpp

```diff
@@ -5215,32 +5215,27 @@ break
          }
       }
 
-   if (!comp()->getOption(TR_DisableSIMDStringCaseConv))
+   bool doSIMDStringCaseConv = cg()->getSupportsSIMDStringCaseConv() && !comp()->getOption(TR_DisableSIMDStringCaseConv);
```

Comment:
Similarly to below comment about `!comp()->getOption(TR_DisableSIMDStringCaseConv)`.